protocol/x/govplus/keeper/msg_server.go

```diff
@@ -15,3 +21,40 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
 }
 
 var _ types.MsgServer = msgServer{}
+
+func (k msgServer) SlashValidator(
+	goCtx context.Context,
+	msg *types.MsgSlashValidator,
```

Comment:
Are we choosing not to implement `ValidateBasic` for `MsgSlashValidator`? Understand this is optional post 0.50, but are we not doing any validation on the fields? For example, `PowerAtInfractionHeight` should be non-negative

Reply:
Implemented this stuff in ValidateBasic